### ISSUE TITLE
Installation fails and abort due to nodejs paths too long

### ISSUE THREAD

**Issue by ypapouin** [winpython/winpython#669]
Installation fails and abort due to nodejs long paths

eg:
C:\Users\Yann\Documents\temp\09-Septembre\pyth\wpy\WinPython64-3.7.0.2\t\n\node_modules\npm\node_modules\libnpx\node_modules\yargs\node_modules\os-locale\node_modules\execa\node_modules\cross-spawn\node_modules\shebang-command\node_modules\shebang-regex\index.js

A workaround was to install directly to C:\WPy-3702

By the way, why nodejs is included ? it is a dependency ?

**Comment by stonebig** [comment]
yes, it's written at the bottom of https://winpython.github.io/ 

(**) For best Winpython 2018-01 and later experience, it is recommended to have Winpython base directory path smaller than 37 characters.

````
example: 
C:\Users\xxxxxxxx\Downloads\WinPython
your example:
C:\Users\Yann\Documents\temp\09-Septembre\pyth\wpy\
````
you see at least 13 characters too long

**Comment by ypapouin** [comment]
Thank you @stonebig  for the information.

- I propose to add a path length check in the installer to warn the user about this possible issue before he click install.
- Maybe adding a checkbox to ignore JupyterLab and NodeJS  install if the user choose a path too long:
  - [x] I understand that this install path is technically too long and that JupyterLab/NodeJS will not be installed

I also found that there is now a way to bypass this limit on Windows 10 since Anniversary Update but it seems that the application must be compiled with this support:
https://superuser.com/questions/1119883/windows-10-enable-ntfs-long-paths-policy-option-missing

**Comment by stonebig** [comment]
I would rather do an nodejs-less binary
